solr/conf/solrconfig.xml

```diff
@@ -285,7 +285,7 @@
 
     <!-- Maximum number of searchers that may be warming in the background
       concurrently.  An error is returned if this limit is exceeded. Recommend
-      1-2 for read-only slaves, higher for masters w/o cache warming. -->
+      1-2 for read-only slaves, higher for mains w/o cache warming. -->
```

Comment:
If we are removing the "master/slave" terminology, this sentence has both and we only touched one. Some quick research found that Drupal has moved to using "primary/replica" for this context. 
https://www.drupal.org/node/2286193